statsmodels/stats/proportion.py

```diff
@@ -123,6 +123,185 @@ def func(qi):
         raise NotImplementedError('method "%s" is not available' % method)
     return ci_low, ci_upp
 
+def multinomial_proportions_confint(counts, alpha=0.05, method='goodman'):
+    '''Confidence intervals for multinomial proportions.
+
+    Parameters
+    ----------
+    counts : array_like of int, 1-D
+        Number of observations in each category.
+    alpha : float in (0, 1), optional
+        Significance level, defaults to 0.05.
+    method : {'goodman', 'sison_glaz'}, optional
```

Comment:
I would've preferred, but chose not to when I saw the methods in [`statsmodels.stats.proportion.proportion_confint`](http://statsmodels.sourceforge.net/stable/generated/statsmodels.stats.proportion.proportion_confint.html) use underscore. Final decision?